std/typecons.d

```diff
@@ -64,67 +69,44 @@ static if (is(T:Object))
 else
     alias RefT = T*;
```

Comment:
Exactly. We'll probably need to work that over as well. And `scoped!T` doesn't work for nested classes because it uses `emplace`, and AFAIK the only way to construct a nested type is with `new`, in that scope (see our previous conversation about this).